tests/ops/integration_tests/saas/test_hubspot_task.py

```diff
@@ -177,12 +218,14 @@ def test_saas_erasure_request_task(
         get_cached_data_for_erasures(privacy_request.id),
         db,
     )
+    config.execution.masking_strict = temp_masking
 
     # Masking request only issued to "contacts" and "subscription_preferences" endpoints
     assert erasure == {
         "hubspot_instance:contacts": 1,
         "hubspot_instance:owners": 0,
         "hubspot_instance:subscription_preferences": 1,
+        "hubspot_instance:users": 1,
     }
 
     connector = SaaSConnector(connection_config_hubspot)
```

Comment:
I know we had this from before but I would prefer if we didn't use the SaaSConnector and instead made these calls using the `requests` library.

Reply:
done!